test/run-test.ps1

```diff
@@ -1,169 +1,62 @@
+#
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
 [cmdletbinding()]
 param(
-    [switch]$UseImageCache,
+    [string]$DotnetInstallDir,
     [string]$Filter,
     [string]$Architecture
```

Comment:
We could default `Architecture` here as `[string]$Architecture="amd64"`

Reply:
Better to make it a validate set. For example, `[ValidateSet("x86","amd64","arm")] `

Reply:
I had it that way to begin with but you can pass "-Architecture" which is exactly what build-and-test.ps1 does.  When this occurs I want to use "amd64".

Reply:
Yes, let's keep it the way it was.